avahi-common/domain.h

```diff
@@ -33,7 +33,7 @@ AVAHI_C_DECL_BEGIN
  * is calculated like this: RFC1034 mandates maximum length of FQDNs
  * is 255. The maximum label length is 63. To minimize the number of
  * (non-escaped) dots, we comprise our maximum-length domain name of
- * four labels � 63 characters plus three inner dots. Escaping the
+ * four labels á 63 characters plus three inner dots. Escaping the
```

Comment:
```suggestion
 * four labels of 63 characters plus three inner dots. Escaping the
```

Reply:
You are right. The semantically correct symbol is à. It's used in German as a preposition when talking about quantities, describing a property of a set of elements.
I'd say the meaning is something like this:
`...four labels, each of them consisting of 63 characters, plus three inner dots.`